packages/synapse-react-client/src/components/SynapseTable/SynapseTableRenderers.tsx

```diff
@@ -172,7 +172,11 @@ export function TableDataColumnHeader(
   // we have to figure out if the current column is a facet selection
   const facetIndex: number = facets.findIndex(
     (facetColumnResult: FacetColumnResult) => {
-      return facetColumnResult.columnName === columnModel?.name
+      return (
+        facetColumnResult.columnName === columnModel?.name &&
+        // Exclude JSON subcolumn facets, since there may be more than one on a given column
+        facetColumnResult.jsonPath == undefined
+      )
```

Comment:
A JSON Column may have multiple enumerated facets--it doesn't really make sense to show them on the column headers as it's currently designed. So for now, don't include the facet control in the table for JSON subcolumn facets.